.pre-commit-config.yaml

```diff
@@ -32,6 +32,12 @@ repos:
     rev: v4.1.0
     hooks:
     -   id: trailing-whitespace
+        # Indentend, multiline markdown comments require leading whitespace to
```

Comment:
out of curiosity, what is this code change doing?

Reply:
I needed to make this change for the pre-commit hook to stop stripping meaningful whitespace from the beginning of a line in my markdown file. In this commit, I'm skipping the trailing-whitespace pre-commit hook for all markdown files.